### PR TITLE
Add remove_assignees method to the client

### PR DESCRIPTION
**Pull request by licatajustin**
**Issue**
Closes https://github.com/octokit/octokit.rb/issues/983

**Context**
GitHub exposes an API to remove assignees from an Issue/Pull Request. This Pull Request takes advantage of that by adding a public method exposed in `Octokit::Client`. 

Please refer to [GitHub's API](https://developer.github.com/v3/issues/assignees/#remove-assignees-from-an-issue) for reference.

**Usage**:

```ruby
Octokit.remove_assignees("octokit/octokit.rb", 23, ["pengwynn", "joeyw"])
```

### CODE REVIEW

**Comment by licatajustin** [comment]
@tarebyte, anything I can do to help get this merged?

File: lib/octokit/client/issues.rb [hunk 1]
```diff
@@ -329,6 +329,19 @@ def issue_timeline(repo, number, options = {})
...
+      # Remove assignees from an issue
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @param number [Integer] Issue number
+      # @param assignees [Array] Assignees to be removed
+      # @return [Sawyer::Resource] Issue
+      # @see https://developer.github.com/v3/issues/assignees/#remove-assignees-from-an-issue
+      # @example Remove assignees "pengwynn" and "joeyw" from Issue #23 on octokit/octokit.rb
+      #   Octokit.remove_assignees("octokit/octokit.rb", 23, ["pengwynn", "joeyw"])
+      def remove_assignees(repo, number, assignees)
```
Comment (tarebyte) [inline lib/octokit/client/issues.rb]: This is missing the normal patter of an `options` hash as the last argument, would you mind adding it?

```ruby
def remove_assignees(repo, number, assignees, options = {})
  # ...
end
```

Comment (licatajustin) [reply lib/octokit/client/issues.rb]: Sounds good. Just updated the commit.

**Comment by licatajustin** [comment]
Let me know if anything else is needed @tarebyte!

File: lib/octokit/client/issues.rb [hunk 2]
```diff
@@ -329,6 +329,19 @@ def issue_timeline(repo, number, options = {})
       def add_assignees(repo, number, assignees, options = {})
         post "#{Repository.path repo}/issues/#{number}/assignees", options.merge({:assignees => assignees})
       end
+
+      # Remove assignees from an issue
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @param number [Integer] Issue number
+      # @param assignees [Array] Assignees to be removed
+      # @return [Sawyer::Resource] Issue
```
Comment (kytrinyx) [inline lib/octokit/client/issues.rb]: We're missing the options hash in the documentation here.

Comment (licatajustin) [reply lib/octokit/client/issues.rb]: Thanks @kytrinyx for the review.

What are you expecting here? The `options` argument doesn't seem very useful outside of consistency efforts with the rest of this library, as [the GitHub API](https://developer.github.com/v3/issues/assignees/#remove-assignees-from-an-issue) doesn't allow for additional parameters.

Comment (kytrinyx) [reply lib/octokit/client/issues.rb]: Unless I'm missing something, the options hash is where you would pass things like custom accept headers.

Comment (licatajustin) [reply lib/octokit/client/issues.rb]: Added.

Comment (kytrinyx) [reply lib/octokit/client/issues.rb]: Appreciate it! 🌷

Comment (kytrinyx) [reply lib/octokit/client/issues.rb]: Dang. I was a bit quick to merge. Still not seeing the options hash—I'll add it in a separate PR.

**Comment by joker67l** [comment]
Tesoro se vuoi scrivimi diretta qua e invia foto qua! Io nn mi iscrivo da nessuna parte😂😂😂Il 23 febbraio 2018 alle 23.13 Katrina Owen <notifications@github.com> ha scritto:@kytrinyx commented on this pull request.In lib/octokit/client/issues.rb:> @@ -329,6 +329,19 @@ def issue_timeline(repo, number, options = {})
       def add_assignees(repo, number, assignees, options = {})
         post "#{Repository.path repo}/issues/#{number}/assignees", options.merge({:assignees => assignees})
       end
+
+      # Remove assignees from an issue
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @param number [Integer] Issue number
+      # @param assignees [Array] Assignees to be removed
+      # @return [Sawyer::Resource] Issue
Dang. I was a bit quick to merge. Still not seeing the options hash—I'll add it in a separate PR.—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by tarebyte** [comment]
This has been released as part of https://github.com/octokit/octokit.rb/releases/tag/v4.9.0

**Pull request merged**